haystack/utils/deepsetcloud.py

```diff
@@ -635,6 +637,79 @@ def _build_workspace_url(self, workspace: Optional[str] = None):
         return self.client.build_workspace_url(workspace)
 
 
+class EvaluationSetClient:
+    def __init__(self, client: DeepsetCloudClient, workspace: Optional[str] = None, index: Optional[str] = None):
+        """
+        A client to communicate with Deepset Cloud evaluation sets and labels.
+
+        :param client: Deepset Cloud client
+        :param workspace: workspace in Deepset Cloud
+
+        """
+        self.client = client
+        self.workspace = workspace
+        self.index = index
+
+    def get_labels(self, index: str, workspace: Optional[str] = None) -> List[Label]:
+        evaluation_set = self._get_evaluation_set(index=index, workspace=workspace)[0]
```

Comment:
Will rename all `index` occurrences with `evaluation_set_name` to be more precise.

Thanks for the error hint, will build it in :+1:

Reply:
Sorry, I confused `Pipeline` with `DeepsetCloudDocumentStore`. I think `DeepsetCloudDocumentStore` should use  `label_index` instead of `evaluation_set_name` as `label_index` is a property of `BaseDocumentStore` already. So with the current implementation we would end up with both properties and `label_index` basically having no purpose. So sorry for my confusion, but let's use `label_index` throughout `DeepsetCloudDocumentStore` again.